tee-worker/litentry/core/mock-server/src/twitter_official.rs

```diff
@@ -15,45 +15,51 @@
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 #![allow(opaque_hidden_inferred_bound)]
 
-use crate::mock_tweet_payload;
+use crate::{UserShieldingKeyType, MOCK_VERIFICATION_NONCE};
+use ita_stf::helpers::get_expected_raw_message;
 use lc_data_providers::twitter_official::*;
-use litentry_primitives::{ChallengeCode, Identity, IdentityString, Web2Network};
-use sp_core::crypto::AccountId32 as AccountId;
+use litentry_primitives::{Identity, IdentityString, Web2Network};
+use sp_core::{sr25519::Pair as Sr25519Pair, Pair};
 use std::{collections::HashMap, sync::Arc};
 use warp::{http::Response, Filter};
 
 pub(crate) fn query_tweet<F>(
 	func: Arc<F>,
 ) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone
 where
-	F: Fn(&AccountId, &Identity) -> ChallengeCode + Send + Sync + 'static,
+	F: Fn(&Sr25519Pair) -> UserShieldingKeyType + Send + Sync + 'static,
 {
 	warp::get()
 		.and(warp::path!("2" / "tweets" / u32))
 		.and(warp::query::<HashMap<String, String>>())
-		.map(move |_tweet_id, p: HashMap<String, String>| {
-			log::info!("query_tweet");
+		.map(move |tweet_id: u32, p: HashMap<String, String>| {
+			println!("query_tweet, tweet_id: {}", tweet_id);
```

Comment:
can we use log instead ?

Reply:
`log` won't print anything.
I haven't looked into it - maybe because it's globally overridden by the sgx version

I know we have a mixture of `println` and `log`, but that's mainly from upstream. They seem to use `println` in non-enclave and `log` in enclave.